v2/cmd/nuclei/issue-tracker-config.yaml

```diff
@@ -17,6 +17,8 @@
 #  project-name: test-project
 #  # issue-label is the label of the created issue type
 #  issue-label: bug
+#  # reuse-issues causes nuclei to reuse issues for results
+#  reuse-issues: false
```

Comment:
We can use the same name for GitHub to adopt a common language. See: https://github.com/projectdiscovery/nuclei/pull/4152/files#diff-6047d61c2dddb0d5668afa37c58c737a199d4609071ee9a423a054ddcacba2a0R36

Reply:
I wasn't aware of the other change. Renamed it to align with Gitlab's implementation :).